recipe/meta.yaml

```diff
@@ -29,13 +29,14 @@ requirements:
     - python
     - pybind11
     - aws-sdk-cpp
-    - pytorch
+    - pytorch =2.0
```

Comment:
Um.... I only see this now, but this is decidedly _not_ how these things should be done.

Reply:
Hi @h-vetinari, this is not actionable feedback; could you explain how things should have been done?

Reply:
Thanks for the clarification! I pinged you on Matrix with a further question to avoid filing up this PR too much.